### DISCUSSION TITLE
I am unable to hide the elements on small screens

### DISCUSSION THREAD

**Discussion by Meghasyam283** [tailwindlabs/tailwindcss#14368] [Help]
<div className="hidden md:flex items-center">
            <Link to={"/"} className="">
              TV Shows &nbsp; &nbsp;
            </Link>
            <Link to={"/"} className="">
              TV Shows &nbsp; &nbsp;
            </Link>
            <Link to={"/history"} class="">
              Search History
            </Link>
          </div>
![s](https://github.com/user-attachments/assets/0417ea4a-a015-49ca-b012-f2b70ee5e759)
![s1](https://github.com/user-attachments/assets/2308b6a2-c60e-4dab-80db-d31aef396829)

**Comment by LinThitHtwe** [comment]
Would you mind showing the source code so we could help you out on that ?
